README.md

```diff
@@ -34,7 +34,18 @@ If all goes well, you will see the following message:
 Genesis transaction written to "/home/user/.xrnd/config/gentx/gentx-f8038a89034kl987ebd493b85a125624d5f4770.json"
 ```
 #### Submit Gentx (optional)
-Submit your gentx in a PR [here](https://github.com/regen-network/testnets) 
+Submit your gentx in a PR [here](https://github.com/regen-network/testnets)
+
+- Clone the repo using
+
+```sh
+git clone https://github.com/regen-network/testnets
+```
+
+- Copy the generated gentx json file to <repo_path>/algradigon-1/gentxs/<your_gentx_file.json>
```

Comment:
There is no gentx folder :) I'd suggest changing to  `<repo_path>/algradigon-1/<your_gentx_file.json>` or add a empty gentx directory with this PR.

Reply:
We cannot commit empty directories in git. May be I can add commit an empty file